### PR TITLE
Enhancement/active todos fetch af

### PR DESCRIPTION
**Pull request by AllisterF1**

### CODE REVIEW

File: src/utils/index.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    const data = await response.json();
+    console.log(data);
+    newUser(data.user.username);
+  } catch (error) {
+    console.log(error);
+  }
+};
+
+// add an active todo to db - needs token auth adding later - placeholder of jwtToken
+export const addtodo = async (jwtToken) => {
```
Comment (ApacheTech) [inline src/utils/index.js]: Allister noted that this was a typo. Function name should be in camelCase.

**Review by ApacheTech — CHANGES_REQUESTED** [review]
One conversation needs addressing. All else is good.

File: src/utils/index.js (hunk 1 above)
Comment (AllisterF1) [reply src/utils/index.js]: camel case typo fixed. thanks.

**Review by ApacheTech — APPROVED** [review]
All good.

**Pull request merged**